ompi/communicator/comm_cid.c

```diff
@@ -383,24 +395,51 @@ static int ompi_comm_ext_cid_new_block (ompi_communicator_t *newcomm, ompi_commu
     pinfo = (pmix_info_t*)darray.array;
     ninfo = darray.size;
 
-    proc_count = newcomm->c_local_group->grp_proc_count;
+    /*
+     * Make sure all processes participating in the PMIx group construct operation
```

Comment:
PLEASE don't do this! The last PMIx version that needed this was years ago in the v3 series, which OMPI does not support. The Standard has been updated to stipulate that order is not guaranteed, so we won't see it return.

All this does is add more overhead for no value.